extensions/reactive-datasource/runtime/src/main/java/io/quarkus/reactive/datasource/runtime/DataSourceReactiveRuntimeConfig.java

```diff
@@ -145,4 +146,10 @@ public class DataSourceReactiveRuntimeConfig {
      */
     @ConfigItem
     public Optional<String> name;
+
+    /**
+     * Other unspecified properties to be passed to the Reactive SQL Client when creating new connections.
```

Comment:
I'm wondering if this should be clarified: technically these options are passed directly to the PG database server each time a new connection is initiated.

Reply:
@Sanne What if we said something like this?

```
Other unspecified properties to be passed through the Reactive SQL Client directly to the database when new connections are initiated.
```

Reply:
yes sounds good! thanks

Reply:
Done. Just pushed an update.